stable/redis/values-production.yaml

```diff
@@ -0,0 +1,229 @@
+## Bitnami Redis image version
+## ref: https://hub.docker.com/r/bitnami/redis/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/redis
+  tag: 4.0.9
+  pullPolicy: IfNotPresent
```

Comment:
should be `Always` since we're using a rolling tag

Reply:
correct: since this is the prod values file, i think we shouldn't be using a rolling tag and pullPolicy will be `IfNotPresent`